Scripts/application.py

```diff
@@ -0,0 +1,112 @@
+            data[person[0]] = {"email": person[1], "assignment ID": person[2]}
+        json_data = json.dumps(data)
+    return json_data
+    
+@app.route('/api/v1/person/<person_id>', methods = ["GET","DELETE"])
+def return_person(person_id: str) -> str:
+    if request.method == "DELETE":
+        with sqlite3.connect(path_to_db) as connection: 
+            cursor = connection.cursor()
+            try:
+                command = 'DELETE FROM people WHERE person_ID=?;'
+                cursor.execute(command, (person_id,))
+                command = 'DELETE FROM forbiddenPairings WHERE santa_ID=? or assignment_ID=?;'
+                cursor.execute(command, (person_id, person_id))
+            except:
+                return('There was an error. Check that the requested ID is valid.')
+        return return_all_people()
+    
+    if request.method == "GET":
+        # returns a json of <person_id>
+        with sqlite3.connect(path_to_db) as connection: 
+            cursor = connection.cursor()
+            try:
+                command = 'SELECT person_ID, email, givingTo_ID FROM people WHERE person_ID=?;'
+                cursor.execute(command, (person_id,))
+                raw_data: List = cursor.fetchall()
+                data: Dict = {raw_data[0][0]:{"email": raw_data[0][1], "assignment ID": raw_data[0][2]}}
+                json_data = json.dumps(data)
+            except:
+                return('There was an error. Check that the requested ID is valid.')
+        return json_data
+    
+@app.route('/api/v1/forbidden_pairing', methods = ["GET","POST"])
+def return_forbidden_pairings():
+    
+    if request.method == "POST":
+        new_giver_constraint = request.get_json()["santa"]
+        new_receiver_constraint = request.get_json()["assignment"]
+        with sqlite3.connect(path_to_db) as connection:
+            cursor = connection.cursor()
+            command = 'SELECT (person_ID) FROM people WHERE email = ?;'
+            cursor.execute(command, (new_giver_constraint,))
+            giver_ID = cursor.fetchone()[0]
+            cursor.execute(command, (new_receiver_constraint,))
+            receiver_ID = cursor.fetchone()[0]
+            command = 'INSERT into forbiddenPairings (santa, santa_ID, assignment, assignment_ID) VALUES (?, ?, ?, ?);'
+            cursor.execute(command, (new_giver_constraint, giver_ID, new_receiver_constraint, receiver_ID))            
+    
+    # returns a json of all forbidden pairings
+    with sqlite3.connect(path_to_db) as connection:
+        cursor = connection.cursor()
+        cursor.execute('SELECT constraint_ID, santa, santa_ID, assignment, assignment_ID FROM forbiddenPairings;')
+        raw_data: List = cursor.fetchall()
+        data: Dict = {}
+        for pair in raw_data:
+            json_data = json.dumps(data)
+            data[pair[0]] = {"santa": pair[1], "santa_ID": pair[2], "assignment": pair[3], "assignment_ID": pair[4]}
+    return json_data

```

Comment:
don't need to return all pairings after a POST. Just return the pairing that was created

Reply:
done